### PR TITLE
ClusterLoader - log which ReplicationControllers have timed-out.

### PR DESCRIPTION
**Pull request by mm4tt**
This will help us debug issues like https://github.com/kubernetes/kubernetes/issues/73461

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go [hunk 1]
```diff
@@ -463,13 +468,16 @@ type objectChecker struct {
 	stopCh    chan struct{}
 	status    objectStatus
 	err       error
+	// namespacedName of the object being checked.
+	namespacedName string
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: I would call it a key.
While this is namespacedName in all current usecase, this is an implementation detail (e.g. in the future we may want to identify objects by UID, or we will have CRDs that aren't namespaced, or sth).

Comment (mm4tt) [reply clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: Good point, done.

File: clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go [hunk 2]
```diff
@@ -463,13 +468,17 @@ type objectChecker struct {
...
-func newObjectCheker() *objectChecker {
+func newObjectChecker(key string) *objectChecker {
 	return &objectChecker{
-		stopCh:    make(chan struct{}),
-		isRunning: true,
-		status:    unknown,
+		stopCh:         make(chan struct{}),
+		isRunning:      true,
+		status:         unknown,
+		key: key,
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: this will fail gofmt

Comment (mm4tt) [reply clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: Doh, always forgetting about gofmt. 
Done.

**Pull request merged**